compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/diagnostics/DiagnosticErrorCode.java

```diff
@@ -316,6 +316,8 @@
     ERROR_DEFAULTABLE_PARAMETER_CANNOT_BE_INCLUDED_RECORD_PARAMETER("BCE664",
             "error.defaultable.parameter.cannot.be.included.record.parameter"),
     ERROR_INCOMPLETE_QUOTED_IDENTIFIER("BCE665", "error.incomplete.quoted.identifier"),
+    ERROR_OPEN_RECORD_CAN_NOT_CONTAIN_RECORD_REST_DESCRIPTOR("BCE666",
```

Comment:
Let's use "cannot" here which is more common and is used in all other error codes.
```suggestion
    ERROR_OPEN_RECORD_CANNOT_CONTAIN_RECORD_REST_DESCRIPTOR("BCE666",
```